zerver/decorator.py

```diff
@@ -352,7 +352,7 @@ def _wrapped_func_arguments(request, *args, **kwargs):
                 # case insensitive per RFC 1945
                 if auth_type.lower() != "basic":
                     return json_error(_("Only Basic authentication is supported."))
-                role, api_key = base64.b64decode(encoded_value).split(":")
+                role, api_key = base64.b64decode(encoded_value.encode('utf-8')).decode('utf-8').split(":")
```

Comment:
This code now reads as fairly confusing with the  `encoded_value` variable name...